pkgs/development/python-modules/urllib3/default.nix

```diff
@@ -23,7 +23,8 @@ buildPythonPackage rec {
   doCheck = false;
 
   checkInputs = [ pytest mock tornado ];
-  propagatedBuildInputs = [ pyopenssl cryptography idna certifi ipaddress pysocks ];
```

Comment:
Do we really want to remove `certifi`? That's not used for Python 2 support, but for having a list of root certificates. I don't know if we bake our own root certificates into Python.

Reply:
Actually, I don't see it imported at all. I see it used to be referenced in the documentation at some point.

https://github.com/urllib3/urllib3/commit/90ee766d9153e2f0def7ba903895d51a86c13b5f

> We install certifi on every version of Python because it's awesome

Reply:
Indeed. So let's just remove it completely.